suite-native/coin-enabling/src/components/NetworkSymbolSwitchItem.tsx

```diff
@@ -40,6 +36,14 @@ const wrapperStyle = prepareNativeStyle<{ isEnabled: boolean }>((utils, { isEnab
     ],
 }));
 
+const wrapperStyle = prepareNativeStyle(_ => ({
+    paddingVertical: 6,
+    paddingHorizontal: 14,
+    gap: 12,
```

Comment:
Would it make sense to use `utils.spacings` for these? At least for the `gap` for which a constant may be used.

Also, is the padding correct according to design? I know if was there before...

Reply:
We have a discussion (or two in slack) for the 12 size and I am in favor of doing this for all occurrences at once.

Reply:
Yep, realized later that it's the one that's currently missing.